ci/terraform/oidc/api-gateway.tf

```diff
@@ -611,7 +611,7 @@ resource "aws_api_gateway_method" "orch_frontend_proxy_method" {
 
 data "aws_cloudformation_stack" "orch_frontend_stack" {
   count = var.orch_frontend_api_gateway_integration_enabled ? 1 : 0
-  name  = "${var.environment}-orch-frontend"
+  name  = replace("${var.environment}-orch-f-deploy", "authdev1", "dev")
```

Comment:
Does the orch frontend sandpit need updating?

Reply:
It does, theres a second PR going up for this.